cmd/edit/machinepool/nodepool.go

```diff
@@ -0,0 +1,164 @@
+package machinepool
+
+import (
+	"os"
+
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	"github.com/openshift/rosa/pkg/interactive"
+	rprtr "github.com/openshift/rosa/pkg/reporter"
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/spf13/cobra"
+)
+
+func editNodePool(cmd *cobra.Command, nodePoolID string, clusterKey string, cluster *cmv1.Cluster, r *rosa.Runtime) {
+	var err error
+
+	// Try to find the node pool
+	r.Reporter.Debugf("Loading machine pool for hosted cluster '%s'", clusterKey)
+	nodePools, err := r.OCMClient.GetNodePools(cluster.ID())
+	if err != nil {
+		r.Reporter.Errorf("Failed to get machine pools for hosted cluster '%s': %v", clusterKey, err)
+		os.Exit(1)
+	}
+
+	var nodePool *cmv1.NodePool
```

Comment:
Can we use this endpoint `/clusters/{id}/node_pools/{node_pool_id}` to fetch a specific node pool?

Reply:
Good point, I used the "machinepool" logic in the same way, and I saw that there was a loop there.
But it doesn't make sense in this context, it can be a direct call.
Sure.